Subject: Add missing manual page options and fix typo
Origin: upstream, commit 1.10.0-6-gff82566
Author: Jaroslav Skarvada <jskarvad@redhat.com>
Date: Fri May 15 13:48:03 2020 +1000

    in Fedora we did man pages check and it seems the new option
    --missing-window is missing from the manual page.  I also added the
    --version option and fixed one typo in the built-in help.

    Signed-off-by: James Cameron <quozl@laptop.org>

--- a/pptp.8
+++ b/pptp.8
@@ -119,6 +119,15 @@
 Default is 100.  Has no effect if test-type is zero.  The result of
 test types 2 and 3 are undefined if this value is less than ten.
 
+.TP
+.B \-\-missing-window <n>
+Enable 'missing window' validation and set packet tolerance
+(300=default, 6000=recommended).
+
+.TP
+.B \-\-version
+Display version.
+
 
 .SH "ROUTING"
 When PPTP is used in conjunction with a default route on top of the
--- a/pptp.c
+++ b/pptp.c
@@ -127,7 +127,7 @@
             "  --test-type <type>	Damage the packet stream by reordering\n"
             "  --test-rate <n>          Do the test every n packets\n"
             "  --missing-window <n>     Enable 'missing window' validation and set packet\n"
-            "                           polerance (300=default, 6000=recommended)\n",
+            "                           tolerance (300=default, 6000=recommended)\n",
 
             version, progname, progname);
     log("%s called with wrong arguments, program not started.", progname);
